.github/workflows/cd-base.yml

```diff
@@ -104,7 +116,8 @@ jobs:
           git config --global user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
           git config --global user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
           git add ./base/Ballerina.toml
-          git commit -m "[Automated] Prepare hl7v2 Base for next dev cycle"
+          git add ./base/Dependencies.toml
```

Comment:
Could have added both files in a single file.

Reply:
U mean in a single line?

Reply:
sorry - yes
git add ./base/Ballerina.toml ./base/Dependencies.toml